app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ArticlesController < ApplicationController
   def index
     @article = Article.all
+    @pagy, @article = pagy(Article)
```

Comment:
what will be the value of these two variables?
why @article is initialized twice?